cleanlab/rank.py

```diff
@@ -652,3 +653,106 @@ def get_outlier_scores(
         return (outlier_scores, knn)
     else:
         return outlier_scores
+
+
+def get_ood_scores(
+    pred_probs: np.array,
+    confident_thresholds: Optional[np.ndarray] = None,
+    labels: Optional[np.ndarray] = None,
+    adjusted_pred_probs: bool = False,
```

Comment:
change argname to `adjust_pred_probs` to match the other functions in `cleanlab.rank`. More generally, try to match the argnames / internal structure of the other existing functions in this file